README.md

```diff
@@ -365,7 +397,7 @@ NOTE: If you're behind a proxy, use `sudo -E` to preserve user environment.
 
 ```bash
 cd tdx/attestation
-sudo ./setup-attestation-guest.sh
+sudo ./setup-guest.sh
```

Comment:
Although the script is "setup-guest.sh", I suggest changing it to "setup-attestation-guest.sh" to be more explicit. What do you think?

Reply:
Agreed - this is now updated in my last commit.